recipes/coreos-installer/recipe.yaml

```
@@ -0,0 +1,82 @@
+schema_version: 1
+
+context:
+  name: "coreos-installer"
+  version: "0.25.0"
+
+package:
+  name: coreos-installer
+  version: ${{ version }}
+
+source:
+  url: https://github.com/coreos/${{ name }}/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 4a62a06c4eb3ec443bb30f135338e034da206793881425fe43f0343ec90649f2
+
+build:
+  number: 0
+  skip:
+    - win
+    - osx
+  script:
+    - export CARGO_HOME="${BUILD_PREFIX}/.cargo"
+    - export OPENSSL_DIR="${PREFIX}"
+    - export OPENSSL_STATIC=0
+    - export ZSTD_SYS_USE_PKG_CONFIG=0
+    - export CC_x86_64_unknown_linux_gnu="${CC}"
+    - export CXX_x86_64_unknown_linux_gnu="${CXX}"
+    - export AR_x86_64_unknown_linux_gnu="${AR}"
+    - export CFLAGS_x86_64_unknown_linux_gnu="${CFLAGS}"
+    - export CXXFLAGS_x86_64_unknown_linux_gnu="${CXXFLAGS}"
+    - export LDFLAGS_x86_64_unknown_linux_gnu="${LDFLAGS}"
+    - cargo build --release --features rdcore
+    - mkdir -p $PREFIX/bin
+    - cp target/x86_64-unknown-linux-gnu/release/coreos-installer $PREFIX/bin/coreos-installer
+    - cp target/x86_64-unknown-linux-gnu/release/rdcore $PREFIX/bin/rdcore
+
+requirements:
+  build:
+    - ${{ compiler('rust') }} >=1.84
+    - ${{ compiler('c') }}
+    - ${{ compiler('cxx') }}
+    - ${{ stdlib('c') }}
+    - cargo-bundle-licenses
+    - cmake >=3.10
+    - make >=4.0
+    - pkg-config >=0.29
+  host:
+    - openssl >=1.1.1,<4
+    - zlib >=1.2.11
+    - zstd >=1.4.0,<2.0
```

```suggestion
  build:
    - ${{ compiler('rust') }}
    - ${{ compiler('c') }}
    - ${{ compiler('cxx') }}
    - ${{ stdlib('c') }}
    - cargo-bundle-licenses
    - cmake
    - make
    - pkg-config
  host:
    - openssl
    - zlib
    - zstd
```
None of these version pins are necessary as they versions we use by default already newer than specified versions.  Furthermore, please avoid specifying versions for packages in https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml in the recipe itself.  

When these versions do need to be overwritten (typically to use an older version than our current default when it can be justified to do so), the versions should be specified in `variants.yaml`, not in `recipe.yaml`.  This allows our bot to keep track of what version those dependencies are on for a recipe and allows us to automate rebuilds when the ABI version of dynamically linked dependencies changes.